### ISSUE TITLE
Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-qPxE1QbwSH: Connection refused

### ISSUE THREAD

**Issue by davidak** [NixOS/nixpkgs#105299]
**Describe the bug**
Some applications seem to depend on this socket that does not exist.

```
[davidak@gaming:~]$ gnome-disks

(gnome-disks:20836): dbind-WARNING **: 22:33:38.492: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-qPxE1QbwSH: Connection refused

[davidak@gaming:~]$ io.elementary.code --version
[WARNING 22:33:46.720019] [dbind] Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-qPxE1QbwSH: Connection refused
Code 3.4.1

[davidak@gaming:~]$ ll /tmp/dbus-qPxE1QbwSH
ls: cannot access '/tmp/dbus-qPxE1QbwSH': No such file or directory

[davidak@gaming:~]$ systemctl --user show-environment -l | grep DBUS
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
```

The dbus deamon is running:

```
[root@gaming:~]# systemctl status dbus.socket
● dbus.socket - D-Bus System Message Bus Socket
     Loaded: loaded (/nix/store/f7xpkxhkn06yi09zd8a3wj1k915i4nmd-dbus-1.12.20/etc/systemd/system/dbus.socket; linked; vendor preset: enabled)
     Active: active (running) since Thu 2020-11-12 04:06:36 CET; 2 weeks 2 days ago
   Triggers: ● dbus.service
     Listen: /run/dbus/system_bus_socket (Stream)
     CGroup: /system.slice/dbus.socket

Nov 12 04:06:36 gaming systemd[1]: Listening on D-Bus System Message Bus Socket.
```

Reproducible with:

- gnome-disks
- gnome-font-viewer
- gnome-mahjongg
- gnome-system-monitor
- gnumeric
- gajim
- io.elementary.calculator
- io.elementary.calendar
- io.elementary.camera
- io.elementary.code
- io.elementary.files
- io.elementary.music
- io.elementary.photos
- io.elementary.screenshot-tool
- io.elementary.switchboard
- io.elementary.videos
- darktable
- dino
- fractal
- meld
- mumble
- com.github.philip-scott.notes-up
- remmina
- obs
- virt-manager

It seem all GNOME/elementary desktop apps, maybe all GTK apps?

Not reproducible with

- gimp
- audacity
- kdenlive
- klavaro

**To Reproduce**
1. Run `gnome-disks` for example
2. Get error (warning): Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-qPxE1QbwSH: Connection refused
3. The application starts and is usable. Not sure which consequences the message has

**Expected behavior**
No errors when applications are started from terminal

**Additional context**
I'm using the Pantheon desktop. This is the complete config: https://codeberg.org/davidak/nixos-config/src/commit/8a839a6d03ff705c3698dc01187757b4a8f37abe/machines/gaming/configuration.nix

**Notify maintainers**
@worldofpeace @jtojnar do you have an idea what this is?

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.8.18, NixOS, 20.09.1721.896270d629e (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.7`
 - channels(root): `"home-manager-20.09, nixos-20.09.1721.896270d629e, nixos-hardware, nixos-unstable-21.03pre251123.b839d4a8557"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by hissssst** [comment]
Hi @davidak, have you managed to fix this error?
Have you experimented with `dbus-run-session` ?

**Comment by davidak** [comment]
> Hi @davidak, have you managed to fix this error?

I don't had the issue in some time. I think a reboot fixes it.

> Have you experimented with dbus-run-session ?

No
